api/AudioTrackList.json

```diff
@@ -783,7 +783,7 @@
               ]
             },
             "ie": {
-              "version_added": "10"
+              "version_added": "11"
```

Comment:
This will be out of sync with the `removetrack` event entry now. Can you check if the event itself is fired in IE10 and IE11? Was it just the `onremovetrack` event handler attribute that was missing in IE10?

Reply:
Note to self: this change was backed out.